### PR TITLE
New Councillor Job Titles

### PR DESCRIPTION
**Pull request by DukeJeans**
## Changelog
* Wrote new job titles for councillors individualized for every in-code culture group, excluding spiritual councillors.

### CODE REVIEW

**Comment by geffe1** [comment]
ck2_474
Male siren
image


Treasurer perhaps? Instead, for the trolls?
Reference: https://wow.gamepedia.com/Royal_Treasury
ck2_471


Quilboar is more or less vanilla..?
ck2_472


Other than that, it all seems ok.

File: localisation/00_wc_job_titles.csv [hunk 1]
```diff
@@ -63,3 +41,316 @@ job_spiritual_void_group;Shadow Priest;Pr
...
+job_spymaster_ancient_group;Ancient of Shadows;Ancien des Ombres;Urtum der Schatten;;;;;;;;;;;x
+job_spymaster_female_ancient_group;Ancient of Shadows;Ancienne des Ombres;Urtum der Schatten;;;;;;;;;;;x
+job_chancellor_aqir_group;Vizier;Vizir;Wesir;;;;;;;;;;;x
+job_chancellor_female_aqir_group;Vizier;Vizir;Wesirin;;;;;;;;;;;x
+job_marshal_aqir_group;Blade Lord;Ma�tre des Lames;Meister der Klingen;;;;;;;;;;;x
+job_marshal_female_aqir_group;Blade Lady;Ma�tre des Lames;Meisterin der Klingen;;;;;;;;;;;x
+job_treasurer_aqir_group;Swarmspeaker;;Herold;;;;;;;;;;;x
+job_treasurer_female_aqir_group;Swarmspeaker;;Heroldin;;;;;;;;;;;x
+job_spymaster_aqir_group;Emissary;Emissaire;Gesandter;;;;;;;;;;;x
+job_spymaster_female_aqir_group;Emissary;Emissaire;Gesandte;;;;;;;;;;;x
```
Comment (zumbak04) [inline localisation/00_wc_job_titles.csv]: Emissary title is used by Chancellors. I suggest Saboteur.
https://wow.gamepedia.com/Saboteur_Kip%27tilak

File: localisation/00_wc_job_titles.csv [hunk 2]
```diff
@@ -63,3 +41,316 @@ job_spiritual_void_group;Shadow Priest;Pr
...
+job_spymaster_draenei_group;Rangari Prime;;Erster Rangari;;;;;;;;;;;x
+job_spymaster_female_draenei_group;Rangari Prime;;Erste Rangari;;;;;;;;;;;x
+job_chancellor_dragon_group;Broodlord;;Brutmeister;;;;;;;;;;;x
+job_chancellor_female_dragon_group;Broodlady;;Brutmeisterin;;;;;;;;;;;x
+job_marshal_dragon_group;General;;Kommandeur;;;;;;;;;;;x
+job_marshal_female_dragon_group;General;;Kommandeurin;;;;;;;;;;;x
+job_treasurer_dragon_group;Majordomo;;Majordomus;;;;;;;;;;;x
+job_treasurer_female_dragon_group;Majordomo;;Majordomus;;;;;;;;;;;x
+job_spymaster_dragon_group;Emissary;;Gesandter;;;;;;;;;;;x
+job_spymaster_female_dragon_group;Emissary;;Gesandte;;;;;;;;;;;x
```
Comment (zumbak04) [inline localisation/00_wc_job_titles.csv]: Emissary title is used by Chancellors. I suggest Saboteur.
https://wow.gamepedia.com/Azure_Saboteur

File: localisation/00_wc_job_titles.csv [hunk 3]
```diff
@@ -63,3 +41,316 @@ job_spiritual_void_group;Shadow Priest;Pr
...
+job_spymaster_pygmy_group;Scout;;;;;;;;;;;;;x
+job_spymaster_female_pygmy_group;Scout;;;;;;;;;;;;;x
+job_chancellor_quillboar_group;Speaker;;;;;;;;;;;;;x
+job_chancellor_female_quillboar_group;Speaker;;;;;;;;;;;;;x
+job_marshal_quillboar_group;Warcaller;;;;;;;;;;;;;x
+job_marshal_female_quillboar_group;Warcaller;;;;;;;;;;;;;x
+job_treasurer_quillboar_group;Kraulshaper;;;;;;;;;;;;;x
+job_treasurer_female_quillboar_group;Kraulshaper;;;;;;;;;;;;;x
+job_spymaster_quillboar_group;Scout;;;;;;;;;;;;;x
+job_spymaster_female_quillboar_group;Scout;;;;;;;;;;;;;x
```
Comment (zumbak04) [inline localisation/00_wc_job_titles.csv]: `quilboar_group` instead of `quillboar_group`.

File: localisation/00_wc_job_titles.csv [hunk 4]
```diff
@@ -63,3 +41,316 @@ job_spiritual_void_group;Shadow Priest;Pr
...
+job_spymaster_sethrak_group;Shadow;;;;;;;;;;;;;x
+job_spymaster_female_sethrak_group;Shadow;;;;;;;;;;;;;x
+job_chancellor_shathyar_group;Halahs'uhnish;;;;;;;;;;;;;x
+job_chancellor_female_shathyar_group;Halahs'uhnish;;;;;;;;;;;;;x
+job_marshal_shathyar_group;Phquathi'agthu;;;;;;;;;;;;;x
+job_marshal_female_shathyar_group;Phquathi'agthu;;;;;;;;;;;;;x
+job_treasurer_shathyar_group;Vorzz'uull;;;;;;;;;;;;;x
+job_treasurer_female_shathyar_group;Vorzz'uull;;;;;;;;;;;;;x
+job_spymaster_shathyar_group;Gaz'qor'magg;;;;;;;;;;;;;x
+job_spymaster_female_sethrak_group;Gaz'qor'magg;;;;;;;;;;;;;x
```
Comment (zumbak04) [inline localisation/00_wc_job_titles.csv]:
```suggestion
job_spymaster_female_shathyar_group;Gaz'qor'magg;;;;;;;;;;;;;x
```

**Comment by geffe1** [comment]
Everything is ok... except for male sirens.

**Pull request merged**